sansa-rdf-flink/src/main/scala/net/sansa_stack/rdf/flink/stats/RDFStatistics.scala

```diff
@@ -52,25 +54,25 @@ class RDFStatistics(rdfgraph: RDFGraph, env: ExecutionEnvironment) extends Seria
 }
 
 object RDFStatistics {
-  def apply(rdfgraph: RDFGraph, env: ExecutionEnvironment) = new RDFStatistics(rdfgraph, env) //.run()
+  def apply(rdfgraph: RDFGraph, env: ExecutionEnvironment): RDFStatistics = new RDFStatistics(rdfgraph, env)
 }
 
 class Used_Classes(rdfgraph: RDFGraph, env: ExecutionEnvironment) extends Serializable with Logging {
```

Comment:
Here and in classes further below there are method names in upper camel case. Since you already changed this in the Spark version, I'd suggest to also change it here.

Reply:
Yep, I just provided implicits for stats instead of using these methods. 
Many thanks for your feedback!